doc/effects.md

```diff
@@ -46,6 +46,24 @@ square.addEffect(ScaleEffect(
 ));
 ```
 
+## RotationalEffect
+
+Applied to `PositionComponent`s, this effect can be used to rotate the component, using an [animation curve](https://api.flutter.dev/flutter/animation/Curves-class.html).
+
+Usage example:
+```dart
+import 'dart:math';
+
+import 'package:flame/effects/effects.dart';
+
+// Square is a PositionComponent
+square.addEffect(RotationalEffect(
+  rotation: 2 * pi, // In radians
```

Comment:
Just a thought, I always find it easier to think in angle instead of radians, I would leave for the effect to converte the angle to radians, what do you think?

Reply:
I actually find it much easier to use radians haha. I think we should first and foremost stick to conventions, angles are used in a lot of places already in flame and I don't recall now which unit is used (I probably copied flutter's default). So I think we should use that one, whichever it is :)

Reply:
PositionComponent#angle is in radians.

I think angles is way more human friendly, as I can easily visualize what 90° degres mean, when any radian value means nothing to me haha

but if we are keeping with the standards, which I think we should, lets keep in radians.

@spydon lets use this PR to add a comment on the angle property of PositionComponent making it clear that that property is in radians.

Reply:
I think it is much better to teach people radians if they haven't already been taught it in school. Angle has no real basis, meanwhile radians do.
It also feels like radians are the standard for most programming libraries, that I have used at least.
But you are correct Erick, it is very much a thing to get used to. :)